docs/documentation/telemetry.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: documentation
+title: Telemetry
+doc_order: 100
+description: About the use of Telemetry in mozregression
+---
+
+# Telemetry
+
+Unless explicitly disabled, mozregression will send an anonymized telemetry ping (a small JSON-document) to Mozilla's servers on each use (command-line invocation in the case of console mozregression, each bisection or single run in the case of the GUI).
+
+This data is anonymized and not personally identifiable, it contains only:
+
+- A random client id (e.g. `5a26b9f7-c025-4858-abce-63ca73ceebf0`)
+- Some very basic information on the environment in which mozregression
```

Comment:
```suggestion
- [Minimal information](https://mozilla.github.io/glean/book/user/pings/index.html#ping-sections) on the environment in which mozregression
```

Reply:
I think glean_parser should do a better job at documenting that pings also contain some information about the device. I filed [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1625412) to improve our docs so that you don't have to mention it.

Reply:
Great, I'll just take this out and refer people to the autogenerated docs